addon-test-support/fire-event.js

```diff
@@ -99,6 +99,10 @@ function buildMouseEvent(type, options = {}) {
       eventOpts.metaKey,
       eventOpts.button,
       eventOpts.relatedTarget);
+
+    if (eventOpts.target !== undefined) {
```

Comment:
This interesting. I'd have expected that firing the event on an element would set the target of the event of that element. Isn't that the case?

if it's not, is there any case in which the target is different from the element in which the event is being fired on? Because if it's always that element, I'd set it every time, not only if the user passes an option.